controller/rest/crdsecurityrule.go

```diff
@@ -3358,6 +3358,15 @@ func handlerGroupCfgExport(w http.ResponseWriter, r *http.Request, ps httprouter
 
 		// export process and file profiles
 		exportProcessRule(gname, &rconf, &(resptmp.Spec), acc)
+		if utils.DoesGroupHavePolicyMode(gname) {
+			allowed := utils.NewSetFromStringSlice([]string{share.PolicyModeLearn, share.PolicyModeEvaluate, share.PolicyModeEnforce})
```

Comment:
It's better to define valid policies as a constant. It doesn't look like we will change it from time to time.

Reply:
share.PolicyModeLearn, share.PolicyModeEvaluate, share.PolicyModeEnforce are string constant